src/components/VKButton.tsx

```diff
@@ -0,0 +1,23 @@
+import getVKCode from "../helpers/parseUrl";
+import { REDIRECT_URL, VK_APP, BACKEND_URL } from "../constants";
+
+const VKButton = () => {
+  const handleRedirect = () => {
+    window.location.href = `https://oauth.vk.com/authorize?client_id=${VK_APP.id}&display=popup&redirect_uri=${REDIRECT_URL}&scope=email&response_type=code&v=5.120&state=4194308`;
+    const code = getVKCode(window.location.href);
+    if (code) {
+      console.log(code);
+    }
+    else {
+      window.location.href = BACKEND_URL;
```

Comment:
You can use navigate().

Reply:
Hmm, I'm sure, that now it's the better implementation (https://stackoverflow.com/questions/948227/should-i-use-window-navigate-or-document-location-in-javascript)

If you mean some of react-router methods - it's not possible, because backend is not related to the our react app